src/main/java/org/medicmobile/webapp/mobile/Alert.java

```diff
@@ -1,20 +1,42 @@
 package org.medicmobile.webapp.mobile;
 
+import static android.os.VibrationEffect.DEFAULT_AMPLITUDE;
+
+import android.annotation.TargetApi;
 import android.content.Context;
 import android.media.MediaPlayer;
+import android.os.Build;
+import android.os.VibrationEffect;
 import android.os.Vibrator;
+import android.os.VibratorManager;
 
 public class Alert {
-	private final MediaPlayer m;
-	private final Vibrator v;
+	private final MediaPlayer mediaPlayer;
+	private final Vibrator vibrator;
+
+	@SuppressWarnings("deprecation")
```

Comment:
spotbugs lint still complains about the deprecated code even though we're already handling it with `Build.VERSION.SDK_INT >= XX` condition.